### DISCUSSION TITLE
Forks lowlighter/lowlighter

### DISCUSSION THREAD

**Discussion by jackdroach** [lowlighter/metrics#605] [Q&A] [answered]
My workflow forks https://github.com/lowlighter/lowlighter on each run. Is this a bug or a feature? Is there any way I can disable it?

Update: After tinkering with random stuff for a while it has stopped. I still don't know what caused it tho

**Comment by lowlighter** [comment, marked as answer]
Do you have an old workflow run you could link which had this behaviour ?
There is nothing in this action which is supposed to do this so if that's the case it's definitely a bug
